lua/lspconfig/haxe_language_server.lua

```diff
@@ -0,0 +1,61 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.haxe_language_server = {
+  default_config = {
+    cmd = {"haxe-language-server"};
+    filetypes = {"haxe"};
+    root_dir = util.root_pattern("build.hxml");
+    settings = {
+      ["haxe.executable"] = "haxe";
+    };
+    init_options = {
+      displayArguments = {"build.hxml"};
+    };
+  };
+  docs = {
+    description = [[
+https://github.com/vshaxe/haxe-language-server
+
+Though bundled with the Visual Studio Code Haxe Extension, the Haxe
+language server can be built as a standalone project by cloning the
+repo and then running the following commands from the project's root
+directory:
+
+    npm install
+    npx lix run vshaxe-build -t language-server
+
+This will create a JavaScript file at `bin/server.js` that can be
+executed with `node`. Note that the server requires Haxe 3.4.0 or
+newer.
+
+The default configuration for the Haxe language server expects an
```

Comment:
I'm almost of the mind to suggestcutting the section on writing a shell file for previty, and just suggesting people use the 
```lua
lspconfig.haxe_language_server.setup({
  cmd = {"node", "path/to/bin/server.js"},
})
``` 

style

Reply:
Good point. I agree.